library/Fission/Web/Heroku.hs

```diff
@@ -75,8 +76,8 @@ provision Request {uuid, region} = do
                        logError <| displayShow err
                        return []
 
-  username <- liftIO <| User.genID
-  secret   <- liftIO <| Random.text 200
+  let username = T.pack . UUID.toString <| uuid
```

Comment:
One character simpler:

```haskell
T.pack <| UUID.toString uuid
```

Reply:
Now that this is pure, we can probably just stick it inline (line 99). Thoughts?

Reply:
Ohh I like the way that reads! Will change.

> Now that this is pure, we can probably just stick it inline (line 99). Thoughts?

We also use it on line 82.